LandSandBoat/server
# [SMN] Correct Fenrir animations & Blood Pact abilities

**_I affirm:_**

## What does this pull request do?



Updates all of Fenrir's animations to reflect new pet_skills sql table
Compared all Blood pact msg's to retail captures
Added new msg enums to reflect retail captures
Removes Fenrir's animation shifting from core
Added SQL updates for future Fenrir Blood Pact abilities

## Steps to test these changes


Use Fenrir's BP Ward to see proper MP consumption and messages displayed
Use Fenrir's BP Rage to see proper MP consumption and messages displayed
Tested animations of future Blood Pact abilities
